doc/time/strftime.rdoc

```diff
@@ -0,0 +1,353 @@
+== Formats for \Method +strftime+
+
+Each of the methods Time#strftime, DateTime#strftime, and Date#strftime
+takes an optional string argument +format+.
+
+The format may have embedded _directives_ that control formatting.
+Non-directive text is copied through to the output string.
+
+About the examples on this page:
+
+- For most directives, the three classes
+  (Time, DateTime, and Date) behave the same way.
+  In those cases examples are given only for Time,
+  whose example instance is named +t+.
+- For a few directives, the behaviors differ.
+  In those cases examples are given for all three classes,
+  whose example instances are named +t+ (\Time),
+  +dt+ (\DateTime), and +d+ (\Date).
+
+A format directive consists of:
+
+- A percent character (<tt>'%'</tt>).
+- Zero or more <em>flags</em>.
+- An optional <em>minimum field width</em>.
+- An optional <em>modifier</em>.
+- A 1-letter conversion specifier.
+
+<b>Minumum Field Width</b>
+
+The default width for a field is the width of the data itself.
+The integer optional minimum field width specifies the minimum width;
+it is ignored if smaller than the default width:
+
+  t = Time.now
+  t.strftime('%Y')   # => "2021"
+  t.strftime('%8Y')  # => "00002021"
+  t.strftime('%20Y') # => "2021"
```

Comment:
Do you mean `%0Y`?

Reply:
Yes, now corrected.  Also added a negative-integer example.